.github/workflows/actions.yml

```diff
@@ -7,14 +7,40 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    # A matrix proves the supported range of Go versions work. This must always
+    # include the floor Go version policy of Mosn and the current Go version.
+    # Mosn's floor Go version for libraries is two behind current, e.g. if Go
+    # supports 1.19 and 1.20, Mosn libraries must work on 1.18, 1.19 and 1.20.
+    #
+    # A floor version is required to ensure Mosn can receive security patches.
+    # Without one, dependencies become locked to an old version of Go, which
+    # itself receives no security patch updates.
+    #
+    # Mosn's binary is built with Go's floor version, e.g. if Go supports 1.19
```

Comment:
@taoyuanyuan fyi integrated this into the comments https://github.com/mosn/mosn/issues/2153#issuecomment-1298047818